torch/csrc/api/include/torch/nn/init.h

```diff
@@ -36,7 +34,7 @@ TORCH_API double calculate_gain(
 
 /// Fills the given `tensor` with the provided `value` in-place, and returns it.
 /// No gradient will be recorded for this operation.
-TORCH_API Tensor constant_(Tensor tensor, Scalar value);
+TORCH_API Tensor constant_(const Tensor& tensor, const Scalar& value);
```

Comment:
Aren't all these changes BC-breaking changes for our public C++ API?

Reply:
A more restrictive constraint is applied to the parameters, the semantics are not changed.